include/mbedtls/bn_mul.h

```diff
@@ -48,7 +48,11 @@
 /* armcc5 --gnu defines __GNUC__ but doesn't support GNU's extended asm */
 #if defined(__GNUC__) && \
     ( !defined(__ARMCC_VERSION) || __ARMCC_VERSION >= 6000000 )
-#if defined(__i386__)
+/*
+ * The i386 assembly is reported to be broken.
+ * Disable it for now, until we're able to fix it.
+ */
```

Comment:
There isn't a bug report... 
I discovered it, running all.sh script, on my 32 bit linux VM.